### PR TITLE
Part4

### PR DESCRIPTION
**Pull request by david-beauchesne**
I think the markdown section will need to be shorter to fit in the time, and I would like to add resources at the end for github pages

### CODE REVIEW

File: _04_rmarkdown.Rmd [hunk 1]
```diff
@@ -12,14 +12,243 @@ output:
...
+.pull-right[
+## `git clone` 
+
+- Clone repo 
+- Make changes 
+- Push / merge changes
+
+.font80[`r nf()` Consider using *Pull requests* when collaborating on a repo (next session)]
+
+.font80[`r nf()` must be a collaborator or organization member to push commits to a repo you do not own]
```
Comment (KevCaz) [inline _04_rmarkdown.Rmd:155]: Maybe worth saying that proper right are required (no need to add that in the slide IMHO)

File: _04_rmarkdown.Rmd [hunk 2]
```diff
@@ -12,14 +12,243 @@ output:
...
+
+.pull-right[
+## `git clone` 
+
+- Get a local copy of your `r gh()` repo 
+- Actively collaborate on a projects
+
+.font80[`r tr()` Cloning is still required if forking a `r gh()` repo]
+]
+
```
Comment (KevCaz) [inline _04_rmarkdown.Rmd:196]: You said that you read a good post about this, maybe worth adding the ref here?

File: _04_rmarkdown.Rmd [hunk 3]
```diff
@@ -143,7 +381,7 @@ Paragraph
```
Comment (KevCaz) [inline _04_rmarkdown.Rmd:381]: BTW Aaron Schwartz ==> The Internet's Own Boy

**Review by KevCaz — APPROVED** [review]
Looks good to me, good job! I have some minor edits here and there + some stuff related to the deployment that I'll push in a sec. 
Don't forget to add ref (slide 39/56)

File: _04_rmarkdown.Rmd (hunk 1 above)
Comment (david-beauchesne) [reply _04_rmarkdown.Rmd:155]: @KevCaz what do you mean that is different than what is not already written there? Do you simply mean that I should state that in order to be a collaborator or an organization member, you need to have been added as such? Or do you mean something else?

**Pull request merged**